readme.txt

```diff
@@ -102,6 +102,23 @@ To avoid this warning and prevent a refresh from altering an edited record it's
 
 Your meeting list records will now no longer display a warning message when being edited, and will not be overwritten by a data source refresh operation!
 
+= How to fix the same meeting appearing repetitively in the "Data Source Changes Detected" email reports? =
+  "Data Source Changes Detected" email reports are sent to listed Change Notification Email receipients following the daily scheduled Change Detection scan. You may notice the same meeting(s) in the data source import   
+  are repeatedly listed as being out of sync with your website meeting list details. The data source feed in question may be refreshed successfully, per the instructions in the email. Yet there it is again the next day! 
+  How to resolve this problem so that the same meeting(s) don't appear in the email report every day? 
+
+  The most common cause of this repetive differences being detected comes during our Google API address validation process which ensures only those location addresses that Google can find are stored locally in your website 
+  database. Quite often Google changes the entered address information into a format that works for them, and that's what gets stored for your meeting in the local database field. We provide a debug utility to disply the 
```

Comment:
tiny typo here `disply` - as to the larger issue here though i think maybe we need to store the address of the imported feed (pre-geocode) so we dont' actually alert the user on these geocoding changes

Reply:
My proofing tool is erratic. Will give it a kick. We will discuss...